ledger/transaction.ts

```diff
@@ -155,12 +153,11 @@ export async function createNativeSegwitPsbt(
   return psbt;
 }
 
-export function addSignitureToStxTransaction(transaction: string | Buffer, signatureVRS: Buffer) {
-  const deserialzedTx = deserializeTransaction(transaction);
+export function addSignatureToStxTransaction(transaction: string | Buffer, signatureVRS: Buffer) {
```

Comment:
did we grep the codebases to see where this is being used, and open PRs to update?

one way to decouple the releases of xverse-core and consuming apps, would be to duplicate the function, and mark the old one with JSDocs: @deprecated comment. then we can be more flexible

Reply:
Agree, although in this particular case the function is ledger-specific and is only used in `ledger/index.ts` in core repo currently, so I believe that it should be safe to simply rename it in scope of this PR